### PR TITLE
Nuri/Update: update Paraguay address and other related styling issue

### PR DESCRIPTION
**Pull request by njazuli-deriv**
Changes:

-   Update Paraguay office image,address and links
-   Update styling to match the design (both desktop and mobile design)

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [X] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [X] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by nikolaiberestevich-deriv — APPROVED** [review]
LGTM  : )

**Review by suthesan-deriv — DISMISSED** [review]

**Review by suthesan-deriv — DISMISSED** [review]

File: src/pages/contact_us/_offices.js [hunk 1]
```diff
@@ -357,11 +400,19 @@ export const Offices = () => {
...
                                             rel="noopener noreferrer"
                                         >
-                                            E-5-6, Soho Ipoh 2,
-                                            <br></br>
-                                            Jalan Sultan Idris Shah,
-                                            <br></br>
-                                            Ipoh 30000, Perak
+                                            <Show.Desktop min_width="mobileL">
+                                                E-5-6, Soho Ipoh 2,
+                                                <br></br>
```
Comment (habib-binary) [inline src/pages/contact_us/_offices.js:405]: Hi @njazuli-deriv  Can we remove the br and use aragrap or span tags

**Review by habib-binary — COMMENTED** [review]
Number of BR tags needs to be removed

File: src/pages/contact_us/_offices.js (hunk 1 above)
Comment (habib-binary) [reply src/pages/contact_us/_offices.js:405]: Cool, we can create a separate card for this.

**Review by habib-binary — APPROVED** [review]
~ LGTM

**Pull request merged**